variables.tf

```diff
@@ -162,3 +162,9 @@ variable "replicas" {
   default     = []
   description = "List of regions to create replica"
 }
+
+variable "enable_tags" {
```

Comment:
```suggestion
variable "tags_enabled" {
```

Reply:
https://docs.cloudposse.com/reference/best-practices/terraform-best-practices/#use-positive-variable-names-to-avoid-double-negatives

Reply:
@osterman updated var name and references